### PR TITLE
New profile: syncthing

### PR DESCRIPTION
**Pull request by gcb**
https://docs.syncthing.net/

tested with v1.28.0

### CODE REVIEW

**Comment by gcb** [comment]
previously https://github.com/netblue30/firejail/issues/295

File: src/firecfg/firecfg.config [hunk 1]
```diff
@@ -851,6 +851,7 @@ surf
 sushi
 swell-foop
 sylpheed
+syncthing
```
Comment (kmk3) [inline src/firecfg/firecfg.config:854]:
```suggestion
#syncthing # server
```

See:

* #6533

File: etc/profile-m-z/syncthing.profile [hunk 2]
```diff
@@ -0,0 +1,116 @@
+# Firejail profile for syncthing
+# Description: File synchronization using public networks
+# This file is overwritten after every install/update
+
+##quiet
```
Comment (kmk3) [inline etc/profile-m-z/syncthing.profile:5]:
```suggestion
```

Remove `##` lines and other noise like in #6549.

Same for the rest of the file.

File: etc/profile-m-z/syncthing.profile [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+#include disable-write-mnt.inc # we set disable-mnt
+#include disable-x11.inc # this causes an error...
+include disable-xdg.inc
+
+
+mkdir ${HOME}/.local/state/syncthing
+whitelist ${HOME}/.local/state/syncthing
```
Comment (kmk3) [inline etc/profile-m-z/syncthing.profile:45]: Is this program intended to be executed at the user or system level?

Does the `syncthing` executable act as the client, server or both?

**Review by kmk3 — CHANGES_REQUESTED** [review]

File: etc/profile-m-z/syncthing.profile (hunk 3 above)
Comment (gcb) [reply etc/profile-m-z/syncthing.profile:45]: Answer is both for both questions.

Ideally it is a user service. Most commonly it is system service. But upstream likes everyone to just run the program in a terminal and then minimize it.

Then it is a peer in a network which will connect to all sorts of things (announce servers, relay servers, other peers), while also exposing servers for LAN auto-discovery and to receive other peer's connections.

**Comment by netblue30** [comment]
Let's go with it, thanks @gcb. I will remove the program from src/firecfg/firecfg.config for now (see kmk3 discussion above).

**Pull request merged**